packages/devops/ci/deployment.Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM amazon/aws-cli
+
+COPY ./packages/devops/scripts/ci/triggerRedeploy.sh .
+
+RUN curl -L https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64 > jq
+RUN chmod +x jq
+RUN mv jq /bin
```

Comment:
This is by no means my area of expertise, but is it possible for us to use docker to install jq like we do for our other dependencies? eg. 
```
# install features not available in base alpine distro
RUN apk --no-cache add \
  bash \
  curl \
  git \
  lastpass-cli \
  openssh \
  postgresql-client \
  rsync \
  jq
```
This post suggests it's possible: https://stackoverflow.com/questions/56452639/how-to-use-httpie-and-jq-within-docker
and there's a docker image for it here: https://hub.docker.com/r/stedolan/jq

Reply:
Yeah I would prefer to be using `apk add` too...unfortunately it seems the aws-cli base docker image strips that out of alpine or something, because `apk` just isn't available. 

Also worth noting that I don't think `apk add` uses docker images under the hood, I'm pretty sure it's more like homebrew on mac and pulls from a repository of binaries.